projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -307,7 +307,7 @@ class_exists( 'Jetpack' ) ||
 					),
 					'hasPaidPlan'                   => $publicize->has_paid_plan(),
 					'isEnhancedPublishingEnabled'   => $publicize->is_enhanced_publishing_enabled( Jetpack_Options::get_option( 'id' ) ),
```

Comment:
Something for a follow-up I should think, but could we similarly sway this check to a `Current_Plan::supports` call?

Reply:
Yeah, sounds good! I have added it here - 1202583289493814-as-1204237958606623